objgraph.py

```diff
@@ -718,25 +734,28 @@ def show_graph(objs, edge_func, swap_source_target,
         dot.wait()
         if dot.returncode != 0:
             # XXX: shouldn't this go to stderr or a log?
-            print("dot failed to generate '%s' image: output format not supported?")
+            print('dot failed to generate "%s" image:',
+                  'output format not supported?')
```

Comment:
I don't think I have `from __future__ import print_statement`, which makes this change a bug.

```
Python 2.7.8 (default, Oct 20 2014, 15:05:19) 
>>> print('dot failed to generate "%s" image:',
...       'output format not supported?')
('dot failed to generate "%s" image:', 'output format not supported?')
```

Now you may ask "whyyyy on Earth do you not use `from __future__ import print_statement`?" and my answer will be "no good reason, I just find __future__ imports a bit ugly."  Despite that, it's better to be ugly than to be buggy, so I'd be +1 for adding the future import.